bloges56/DateSim
# Fd gamemanager relationships

## Description
- Relationship statuses are now being tracked and hearts are showing in an updated scoreboard.

## How to Test
1. Go to SceneManager Scene and hit play
3. Hit start game
4. Enter a name
5. Hold tab to see the scoreboard
6. Play the rhythm game. 
7. After exiting, look at the scoreboard when you're in the arcade. 

## Checklist
- [x] Do you see the name inputted at the top of the scoreboard? 
- [x] Do you see all 3 names for our characters?
- [x] Do you see no hearts next to their names at the start?
- [x] Do you get red hearts if you have a positive relationship status with Remington? 
- [x] Do you get black hearts if you have a negative relationship status with Remington?

## Usage
In order to update relationship statuses in your game:
1. Make sure the gamemanager is accessible in your script. 
2. When you win the game use `gameManager.addRelVal("Deon/Claire/Remington")` with your characters name in the quotes
3. When you lose the game use `gameManager.removeRelVal("Deon/Claire/Remington")` with your characters name in the quotes. 
4. Make sure these functions are only ran once.
5. Only this command is needed to update the values. 
6. If you want to get the relationship value for each character, use `getRelVal("Deon/Claire/Remington")` with your characters name in the quotes. It will return the value as an int.
